web/hooks/organization.go

```diff
@@ -0,0 +1,47 @@
+package hooks
+
+import (
+	"context"
+
+	"github.com/google/go-github/v45/github"
+	"github.com/quickfeed/quickfeed/qf"
+	"github.com/quickfeed/quickfeed/scm"
+)
+
+func (wh GitHubWebHook) handleOrgInvite(e *github.OrganizationEvent) {
+	// check that invitee is enrolled in course
+	invitee := e.GetInvitation().GetLogin()
+	user, err := wh.db.GetUserByCourse(&qf.Course{ScmOrganizationID: uint64(e.GetOrganization().GetID())}, invitee)
+	if err != nil {
+		wh.logger.Error("Invite: Failed to find user:", err)
+		return // course does not exist, or user not enrolled
+	}
+
+	invite := e.GetInvitation()
```

Comment:
This check should be moved first in this method. We shouldn't print `No invite` in the log; I think we can just ignore nil invites. Does this actually happen?